satellite/orders/endpoint.go

```diff
@@ -138,36 +135,11 @@ func (endpoint *Endpoint) Settlement(stream pb.Orders_SettlementServer) (err err
 		}
 
 		rejectErr := func() error {
+			// satellite verifies that it signed the order limit
 			if err := signing.VerifyOrderLimitSignature(ctx, endpoint.satelliteSignee, orderLimit); err != nil {
 				return Error.New("unable to verify order limit")
 			}
 
-			if orderLimit.DeprecatedUplinkId == nil { // new signature handling
-				if err := signing.VerifyUplinkOrderSignature(ctx, orderLimit.UplinkPublicKey, order); err != nil {
```

Comment:
Why did you remove order signature checking?

Reply:
Got it.. got it.. .got it.. i was under the impression that order limit has all the info of actual uplink's order... I looked at it completely wrong. there were two orders sent back to satellite, 1. the original order limit and 2. the uplink order. Thnx... it is clear... we need to have VerifyUplinkOrderSignature() to verify the order limit that uplink sent to Storagenodes, with the public key that satellite sent to uplink as part of the order limit. Got it.. thnx .. hope I got it this time...  👍